java/source/create.rst

```diff
@@ -187,4 +187,40 @@ Array of List
 
 .. _`FieldVector`: https://arrow.apache.org/docs/java/reference/org/apache/arrow/vector/FieldVector.html
 .. _`ValueVector`: https://arrow.apache.org/docs/java/vector.html
-.. _`dictionary-encoding`: https://arrow.apache.org/docs/format/Columnar.html#dictionary-encoded-layout
\ No newline at end of file
+.. _`dictionary-encoding`: https://arrow.apache.org/docs/format/Columnar.html#dictionary-encoded-layout
```

Comment:
Only by readability could be possible to move these lines to end of file

Reply:
Sure thing, let me move them to end and update the PR. Thanks

Reply:
updated